RadarSDKTests/RadarTestutils.m

```diff
@@ -18,4 +18,181 @@ + (NSDictionary *)jsonDictionaryFromResource:(NSString *)resource {
     return jsonDict;
 }
 
++ (NSMutableDictionary *)createTrackParamWithLocation:(CLLocation *_Nonnull)location
```

Comment:
I guess using the default values returned by `radarState` and `radarSettings` is perfectly fine in this case, but can we just put a comment noting that?

Reply:
added function docstring in the header file for this behaviour